app/controllers/reactions_controller.rb

```diff
@@ -73,7 +73,9 @@ def create
     ).first
 
     # if the reaction already exists, destroy it
-    if reaction
+    # NOTE: @citizen Temporarily we ingore vomit reactions, to avoid accidental
```

Comment:
@rhymes @vaidehijoshi This is by no means an elegant solution, but I think I want to keep the bigger refactoring out of the scope of this PR/RFC. What do you think?